src/bootstrap/bootstrap.php

```diff
@@ -26,6 +26,7 @@ function bootstrap( $dir ) {
 	 * Autoload all php files in /src/
 	 */
 	autoload( $dir . '/src' );
+	autoload( $dir . '/src-psr-4' );
```

Comment:
I'm guessing that this new autoload should not be added. Rather each project should define an autoload in its own composer.json. I'm also not sure how to resolve potential issues where some class in /src wants to extend or implement a class in /src-psr-4 ...

Reply:
We should probably just remove this altogether and define it in the default `wp-project` project's app.php file